js/initcontext_test.go

```diff
@@ -131,7 +131,7 @@ func TestInitContextRequire(t *testing.T) {
 			fs := afero.NewMemMapFs()
 			assert.NoError(t, afero.WriteFile(fs, "/file.js", []byte(`throw new Error("aaaa")`), 0o755))
 			_, err := getSimpleBundle(t, "/script.js", `import "/file.js"; export default function() {}`, fs)
-			assert.EqualError(t, err, "Error: aaaa at file:///file.js:2:7(4)")
+			assert.EqualError(t, err, "Error: aaaa\n\tat file:///file.js:2:7(4)\n\tat reflect.methodValueCall (native)\n\tat file:///script.js:1:117(14)\n")
```

Comment:
This `reflect.methodValueCall (native)` is not a great improvement to not having a stack trace at all :sweat_smile: Can you please also add a test that has a stacktrace consisting of purely JS functions?

Reply:
This is technically `script.js`-> `reflect.methodValueCall`->`file.js`